test/swivel/Swivel.sol

```diff
@@ -396,6 +394,17 @@ contract Swivel {
     return true;
   }
 
+  /// @notice Allows the admin to mass approve all compound markets, saving marginal approvals
+  /// @param u The underlying token address
+  /// @param c The compound cToken address
+  function approveCompound(address[] calldata u, address[] calldata c) external onlyAdmin(admin) returns (bool) {
+    for (uint256 i; i < u.length; i++) {
```

Comment:
see, this one is calldata u, not hoisted...

Reply:
Ahhh thats cause this wasnt audited hah, this is my own code for the new approval workflow on deployment (avoiding approvals on every trade)

Reply:
Can/should add the same optimization to it even though were the only ones calling the function